python/revng/api/_capi.py

```diff
@@ -0,0 +1,69 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+import os
+import traceback
+from pathlib import Path
+from typing import List
+
+from cffi import FFI
+from cffi.backend_ctypes import CTypesBackend
+
+
+ORCHESTRA_ROOT_ENV = os.getenv("ORCHESTRA_ROOT")
```

Comment:
`git grep -i orchestra` in revng should give empty results (except for the README).

Reply:
All references to orchestra removed